framework/test/Volo.Abp.Auditing.Tests/Volo/Abp/Auditing/App/EntityFrameworkCore/AbpAuditingTestDbContext.cs

```diff
@@ -27,5 +27,15 @@ public AbpAuditingTestDbContext(DbContextOptions<AbpAuditingTestDbContext> optio
         {
 
         }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+
+            modelBuilder.Entity<AppFullAuditedEntityWithAudited>(entity =>
+            {
+                entity.Property<string>("Shadow");
+            });

```

Comment:
Maybe we can remove those lines, what do you think?

Reply:
Ok, removed